src/datasources/cache/cache.router.ts

```diff
@@ -0,0 +1,249 @@
+import { CacheDir } from './entities/cache-dir.entity';
+
+export class CacheRouter {
+  private static readonly ALL_TRANSACTIONS_KEY = 'all_transactions';
+  private static readonly BACKBONE_KEY = 'backbone';
+  private static readonly BALANCES_KEY = 'balances';
+  private static readonly CHAIN_KEY = 'chain';
+  private static readonly CHAINS_KEY = 'chains';
+  private static readonly COLLECTIBLES_KEY = 'collectibles';
+  private static readonly CONTRACT_KEY = 'contract';
+  private static readonly CREATION_TRANSACTION_KEY = 'creation_transaction';
+  private static readonly DELEGATES_KEY = 'delegates';
+  private static readonly INCOMING_TRANSFERS_KEY = 'incoming_transfers';
+  private static readonly MASTER_COPIES_KEY = 'master-copies';
+  private static readonly MESSAGE_KEY = 'message';
+  private static readonly MODULE_TRANSACTIONS_KEY = 'module_transactions';
+  private static readonly MULTISIG_TRANSACTION_KEY = 'multisig_transaction';
+  private static readonly MULTISIG_TRANSACTIONS_KEY = 'multisig_transactions';
+  private static readonly OWNERS_SAFE_KEY = 'owner_safes';
+  private static readonly SAFE_APPS_KEY = 'safe_apps';
+  private static readonly SAFE_KEY = 'safe';
+  private static readonly TOKEN_KEY = 'token';
+  private static readonly TOKENS_KEY = 'tokens';
+  private static readonly TRANSFERS_KEY = 'transfers';
```

Comment:
I just copied the keys from the current functions. I think at this point we're free to change these keys and maybe normalize them (`-` vs `_` usage for example).

Reply:
Yeah we can change these keys freely (in the current state of the project). We can do it in a separate PR.